### PR TITLE
Add process manager for move in

### PR DESCRIPTION
**Pull request by kft**
## Description
This PR implements a process manager for the move in business process.

Changes:
- Add `MoveInProcessManager`
- Add `MoveInProcessManagerRouter`
- Add EF core entity configuration for `MoveInProcessManager`

### CODE REVIEW

**Review by Korgath — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/Processing/MoveInProcessManagerTests.cs [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+using Energinet.DataHub.MarketRoles.Domain.EnergySuppliers;
+using Energinet.DataHub.MarketRoles.Domain.MeteringPoints;
+using Energinet.DataHub.MarketRoles.Domain.MeteringPoints.Events;
+using Energinet.DataHub.MarketRoles.Domain.SeedWork;
+using Microsoft.EntityFrameworkCore;
+using NodaTime;
+using Xunit;
+
+namespace Energinet.DataHub.MarketRoles.IntegrationTests.Application.MoveIn.Processing
+{
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/Processing/MoveInProcessManagerTests.cs:32]: missing traits

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Tests/Application/MoveIn/Process/MoveInProcessManagerTests.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+using System;
+using System.Linq;
+using Energinet.DataHub.MarketRoles.Application.MoveIn.Processing;
+using Energinet.DataHub.MarketRoles.Domain.MeteringPoints.Events;
+using NodaTime;
+using Xunit;
+
+namespace Energinet.DataHub.MarketRoles.Tests.Application.MoveIn.Process
+{
+    public class MoveInProcessManagerTests
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Tests/Application/MoveIn/Process/MoveInProcessManagerTests.cs:24]: missing traits

**Review by erduna — COMMENTED** [review]
Missing traits on tests, otherwise OK for our current limited scope 👍

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/Processing/MoveInProcessManagerTests.cs (hunk 1 above)
Comment (kft) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.IntegrationTests/Application/MoveIn/Processing/MoveInProcessManagerTests.cs:33]: Fixed :-)

**Review by erduna — APPROVED** [review]

**Review by Korgath — APPROVED** [review]

**Pull request merged**